### ISSUE TITLE
Conductivity probe mounting

### ISSUE THREAD

**Issue by thomafred** [IanTBlack/CTDizzle#2]
Hi there,

Cool project!

How was the conductivity probe mounted? As far as I understand, the probe is wider than the holes in the end cap

**Comment by IanTBlack** [comment]
If you are using an Atlas Scientific conductivity probe you will need to drill one of the 10mm holes on the end cap to a diameter of 12mm. It looks like Atlas Scientific has changed some things within their datasheet since the CTDizzle was last updated, but drilling to 12mm should hold true for both their "legacy" and current conductivity probes (not the industrial version). I would highly recommend sticking with mm. Their new datasheet inaccurately converts 12mm to 0.5in, while their legacy datasheet resolved it to 0.47in.

A low-end 12mm bit will run you about 10 USD. Another alternative would be to drill out with a 7/16" or 15/32" drillbit (11.11, 11.9 mm respectively) and then use a round file to hand fit the probe to the endcap. If you have access to a drill press, that is optimal. For the version in the [user guide](https://github.com/IanTBlack/CTDizzle/blob/main/documentation/CTDizzle.pdf) I used a cordless drill, which seemed to work fine. I think that after you pot the probe with the PVA molds, the epoxy should fill in any excessive slop between the probe and the drilled hole. Be sure to rough up the surface of the probe that will be in contact with the epoxy.

Let me know if I have answered your question.

Cheers,
- Ian
